File: cat.py
#!/usr/local/bin/python3

import sys

if len(sys.argv) > 1:
    with open(sys.argv[1], "r") as file :
        if not file:
            print("File not found")
            exit(1)
        print(file.read())
else:
    print("File not found")
    exit(1)

File: cat.lua
#!/usr/bin/env lua

if #arg > 0 then
    local file = io.open(arg[1], "rb")
    if file then
        local text = file:read("*a");
        print(text);
        file:close();
    else
        print("File not found")
        os.exit(1)
    end
else
    print("File not found")
    os.exit(1)
end